core/drivers/crypto/hisilicon/hisi_qm.c

```diff
@@ -618,15 +645,10 @@ static enum hisi_drv_status qm_sqc_cfg(struct hisi_qp *qp)
 {
 	enum hisi_drv_status ret = HISI_QM_DRVCRYPT_NO_ERR;
 	struct hisi_qm *qm = qp->qm;
-	struct qm_sqc *sqc = NULL;
-	paddr_t sqc_dma = 0;
-
-	sqc = memalign(HISI_QM_ALIGN32, sizeof(struct qm_sqc));
-	if (!sqc)
-		return HISI_QM_DRVCRYPT_ENOMEM;
-
-	sqc_dma = virt_to_phys(sqc);
+	struct qm_sqc *sqc = qm->cfg_xqc.sqc;
+	struct qm_mailbox mb;
```

Comment:
Please initialize this local variable with `= { }` as described in the coding guidelines.